src/WebJobs.Extensions.Http/ClaimsIdentitySlim.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Security.Claims;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    /// <summary>
+    /// Light-weight representation of a <see cref="ClaimsIdentity"/> object.
+    /// </summary>
+    [DataContract]
+    [KnownType(typeof(ClaimSlim))]
+    internal struct ClaimsIdentitySlim
```

Comment:
>ClaimsIdentitySlim [](start = 20, length = 18)

Add a comment that this needs to be kept in-sync with the Easy Auth version of this same code.